ocs_ci/utility/ocs_build.py

```diff
@@ -0,0 +1,66 @@
+"""
+This module is used to return latest OCS internal build for specified OCS
+version.
+"""
+
+import argparse
+import os
+
+from ocs_ci.framework import config
+from ocs_ci.framework.main import load_config
+from ocs_ci.ocs.constants import OCS_VERSION_CONF_DIR
+from ocs_ci.utility.utils import get_latest_ds_olm_tag
+
+
+def init_parser():
+    """
+    Init argument parser.
+
+    Return:
+        object: Parsed arguments
+    """
+
+    parser = argparse.ArgumentParser(
+        description="OCS Internal build version",
+        formatter_class=argparse.RawTextHelpFormatter,
```

Comment:
What is the reason for `RawTextHelpFormatter`? Without that, it will better split the lines based on the terminal width.

Reply:
Just copy paste a base structure of argparser and add argument from some example and didn't notice this. Removed this option now.